recipes/arabic_reshaper/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "arabic_reshaper" %}
+{% set version = "2.0.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 008d71e6f5523833d8f4c8b5fd487d8bbed444c7c9a06665695afdb5ef47f214
+
+build:
+  noarch: python
+  number: 0
+  # Note: --no-deps is currently required due to https://github.com/conda/conda-build/issues/3254
+  # Once resolved, it should be removed.
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - setuptools
+    - configparser
+    - future
+
+test:
+  imports:
+    - arabic_reshaper
+
+about:
+  home: https://github.com/mpcabd/python-arabic-reshaper
+  license: GPL-3.0
+  license_family: GPL
+  license_file: LICENSE
```

Comment:
I saw you added the license file to the recipe, thank you.

Would you be able to also open a PR against the original package to have the license added to the git repo to include the changes to the MANIFEST.in?

Reply:
Thanks, the PR has been filed: https://github.com/mpcabd/python-arabic-reshaper/pull/26